consul/base.py

```diff
@@ -636,6 +636,57 @@ def put(
             return self.agent.http.put(
                 CB.json(), '/v1/kv/%s' % key, params=params, data=value)
 
+        def replace(
+                self,
+                key,
+                value,
+                cas=None,
+                flags=None,
+                acquire=None,
+                release=None,
+                token=None,
+                dc=None):
+            """
+            Replaces *key* to the given *value* if different from what's
+            already stored in KV
+
```

Comment:
can you explain, 
- it's "like" put, but faster if it's already exist,
- do a put if not exist. ( and not fail )